apps/cookbook/src/app/examples/header-example/examples/custom-content-flag.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-header-example-custom-content-flag',
+  template: `<kirby-header [title]="'Title'" subtitle1="Subtitle one" subtitle2="Subtitle two">
+  <div class="custom-section" *kirbyHeaderCustomFlagSection>
```

Comment:
I think we should remove section from the name throughout the code, as we still want to signal via our naming that the custom content should act as a flag - it is not "just another section".

```suggestion
  <div class="custom-section" *kirbyHeaderCustomFlag>
```